### PR TITLE
Add configuration from versions.yaml

### PR DESCRIPTION
**Pull request by jprzychodzen**

### CODE REVIEW

File: versioning/versions/versions.go [hunk 1]
```diff
@@ -52,6 +66,50 @@ func LoadVersions(path string) Spec {
...
 }
 
+func LoadConfig(path, config string) Config {
+	var whole map[string]interface{}
+	err := yaml.Unmarshal(ReadFile(path), &whole)
+	if err != nil {
+		log.Fatalf("error: %v", err)
+	}
+
+	if c, k := whole[config]; k {
```
Comment (huyhg) [inline versioning/versions/versions.go]: replace `k` with the idiomatic `ok`

File: versioning/versions/versions.go [hunk 2]
```diff
@@ -35,14 +37,26 @@ type Spec struct {
 	Versions []Version
 }
 
-func LoadVersions(path string) Spec {
+type Config map[string]string
```
Comment (huyhg) [inline versioning/versions/versions.go]: Add doc for this type

Comment (jprzychodzen) [reply versioning/versions/versions.go]: Done

File: versioning/versions/versions.go (hunk 1 above)
Comment (jprzychodzen) [reply versioning/versions/versions.go]: Done

**Pull request merged**